packages/fields/src/types/CloudinaryImage/ImageBlock.js

```diff
@@ -19,15 +22,24 @@ export class ImageBlock extends Block {
       auxList = createAuxList(auxListKey, {
         fields: {
           // We perform the requires here to avoid circular dependencies
-          image: { type: CloudinaryImage, isRequired: true, adapter },
+          image: { type: requireDefault('./'), isRequired: true, adapter },
+          align: {
+            type: requireDefault('../Select'),
+            defaultValue: 'center',
+            options: ['left', 'center', 'right'],
+          },
```

Comment:
☝️  The `align` field will probably change, and is mostly there right now to show that it all works as expected.